package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/eth-json-rpc-provider",
-  "version": "1.0.1",
+  "version": "1.0.0",
```

Comment:
@legobeat I see that this PR changes the version from 1.0.1 to 1.0.0. While we could do that, since we are publishing 2.0.0 anyway, I wonder if it would be easier to keep it at 1.0.1 and only update the changelog to indicate that this version has been deprecated. We could also deprecate 1.0.1 on NPM itself (using `npm deprecate`). Then we could bump the version to 2.0.0 and release that.